sync-data-to-another-database/plugin/lib/services/exporter.js

```diff
@@ -25,18 +25,18 @@ class Exporter {
     return this.client
       .connect()
       .then(() => {
-        this.context.log.info('[kuzzle-plugin-export-cassandra] Cassandra client connected');
+        this.context.log.info('[kuzzle-plugin-sync-cassandra] Cassandra client connected');
         return Promise.resolve(true);
```

Comment:
In a promise chain, you don't have to create a promise to return an already resolved value.
Here, you may simply `return true;`